ext/standard/proc_open.c

```diff
@@ -475,6 +487,12 @@ static zend_string *get_valid_arg_string(zval *zv, int elem_num) {
 		return NULL;
 	}
 
+	if (elem_num == 1 && ZSTR_LEN(str) == 0) {
+		zend_value_error("First element must contain a non-empty program name");
+		zend_string_release(str);
+		return NULL;
+	}
```

Comment:
Please create different PR for this.

Reply:
test suite did not pass for me without making sure "command" is non-empty.  but that seemed to be a problem in my environment where test_executable wasn't defined,,,